app/ts/components/pages/AddNewAddress.tsx

```diff
@@ -358,16 +381,24 @@ export function AddNewAddress(param: AddAddressParam) {
 						/>
 					</div>
 				</div>
-				<div style = 'padding-left: 10px; padding-right: 10px; margin-bottom: 10px; height: 50px'>
+				<div style = 'padding-left: 10px; padding-right: 10px; margin-bottom: 10px; height: 80px'>
 					{ errorString === undefined ? <></> : <Notice text = { errorString } /> }
-					{ errorString === undefined && incompleteAddressBookEntry.duplicateStatus === 'Duplicates' ? <Notice text = { 
-						`There already exists ${ incompleteAddressBookEntry.duplicateEntry.type === 'addressInfo' ? 'an address' : incompleteAddressBookEntry.duplicateEntry.type} with ${ 'symbol' in incompleteAddressBookEntry.duplicateEntry ? `symbol "${ incompleteAddressBookEntry.duplicateEntry.symbol }" and` : '' } name "${ incompleteAddressBookEntry.duplicateEntry.name }".`
-					 } /> : <></> }
+					{ errorString === undefined && incompleteAddressBookEntry.duplicateStatus === 'Duplicates' ? <>
+						<Notice text = { `There already exists ${ incompleteAddressBookEntry.duplicateEntry.type === 'addressInfo' ? 'an address' : incompleteAddressBookEntry.duplicateEntry.type} with ${ 'symbol' in incompleteAddressBookEntry.duplicateEntry ? `symbol "${ incompleteAddressBookEntry.duplicateEntry.symbol }" and` : '' } name "${ incompleteAddressBookEntry.duplicateEntry.name }".` } />
```

Comment:
```suggestion
						<Notice text = { `An entry (${ incompleteAddressBookEntry.duplicateEntry.type === 'addressInfo' ? 'an address' : incompleteAddressBookEntry.duplicateEntry.type}) with ${ 'symbol' in incompleteAddressBookEntry.duplicateEntry ? `symbol "${ incompleteAddressBookEntry.duplicateEntry.symbol }" and` : '' } name "${ incompleteAddressBookEntry.duplicateEntry.name }" already exists.` } />
```

Reply:
I don't think "An entry already exists." is proper english, you need to have verb in it ("there")